lib/reporters/base.js

```diff
@@ -169,7 +169,14 @@ exports.list = function(failures) {
     // msg
     var msg;
     var err = test.err;
-    var message = err.message || '';
+    var message;
+    if (err.message) {
+      message = err.message;
+    } else if (typeof err.inspect === 'function') {
+      message = err.inspect();
```

Comment:
Another option here would be to call `util.inspect(err)`, but this wouldn't work outside of Node.